packages/open-truss/src/configuration/engine-v1/RenderConfig.tsx

```diff
@@ -123,3 +133,26 @@ function validateComponentProps(
 
   return errors
 }
+
+function setWorkflowSession(id: string): void {
+  localStorage.setItem(`OT-Workflow:${id}`, JSON.stringify({}))
+}
+
+export function getWorkflowSession(
+  id: string,
+): Record<string, string | number> {
+  const session = localStorage.getItem(`OT-Workflow:${id}`) || ''
+  const sess = JSON.parse(session)
+  if (typeof sess === 'object') return sess
+  return {}
+}
+
+export function setWorkflowSessionValue(
+  id: string,
+  key: string,
+  value: string | number,
+): void {
+  const session = getWorkflowSession(id)
+  session[key] = value
+  localStorage.setItem(`OT-Workflow:${id}`, JSON.stringify(session))
+}
```

Comment:
The store for this will need to be configurable right? Maybe add a TODO here or open an issue?

Reply:
Yup definitely. There is very hacky right now and we'll need to buff this out a lot more. Also we'll need to figure out how we want this data to interact or not interact with the url.  

Added `Implement client / server storage mechanism` to the config spec v0.0.2 milestone issue https://github.com/open-truss/open-truss/issues/112